src/index.js

```diff
@@ -158,7 +158,8 @@ async function loader(source, inputSourceMap, overrides) {
     );
   }
 
-  const config = babel.loadPartialConfig(injectCaller(programmaticOptions));
+  const config = await (babel.loadPartialConfigAsync ||
```

Comment:
nit: Can you add a comment here? something like
```js
// babel.loadPartialConfigAsync is available in v7.8.0+
```

Reply:
Done